tempesta_fw/t/functional/helpers/deproxy.py

```diff
@@ -512,6 +552,10 @@ def handle_read(self):
                            '<<<<<\n%s>>>>>'
                            % self.response_buffer))
             raise
+        if len(self.response_buffer) > 0:
```

Comment:
Please check with `test_pairing.py` tests, i expect problems here. Reach me in chat if you catch an error there, we'll discuss  a solution.